package.json

```diff
@@ -47,7 +47,6 @@
 		"format:changed:main": "pretty-quick --branch main",
 		"format:changed:next": "pretty-quick --branch next",
 		"format:repo": "npm run prettier:repo:fix",
-		"generate:upcoming": "flub generate upcoming --releaseGroup client -t minor",
```

Comment:
wouldn't we still want this so release oce can generate the roll up at release time?

Reply:
We don't usually have scripts for release-related activities. The release drivers run `flub` directly or via `pnpm flub`, so no need to define scripts for things the release drivers do.